Core/src/main/java/org/tribuo/transform/TransformationMap.java

```diff
@@ -49,9 +52,9 @@
 public class TransformationMap implements Configurable, Provenancable<ConfiguredObjectProvenance> {
 
     @Config(mandatory = true,description="Global transformations to apply after the feature specific transforms.")
-    private List<Transformation> globalTransformations;
+    private List<Transformation> globalTransformations = new ArrayList<>();
```

Comment:
This doesn't need to be assigned to an ArrayList if it's mandatory. It'll be overwritten by the config system, and it's overwritten in the constructor (though the constructor could do a defensive copy I guess).